verilog/analysis/verilog_linter_configuration.cc

```diff
@@ -334,8 +354,9 @@ absl::Status LinterConfiguration::AppendFromFile(
     // Log warnings and errors
     if (!error.empty()) {
       std::cerr << "Using a partial version from " << config_filename
-                << ". Found the following issues: " << error;
+                << ". Found the following issues: " << error << std::endl;
```

Comment:
Just use `<< "\n"` instead of `<< std::endl`.

Reply:
There's a couple of other places in this file with std::endl streaming to std::cerr. Do you want me to fix those while I'm there?

Reply:
Let's leave the other places for some separate cleanup PR. That way it is easy to focus on one thing at a time.